augeas/__init__.py

```diff
@@ -57,7 +57,7 @@ def enc(st):
     if st:
         return st.encode(AUGENC)
     else:
-        return ''
+        return b''
 
 
 def dec(st):
```

Comment:
Hmm, I guess that `dec()` should have the same logic. Please see `return ''` below.